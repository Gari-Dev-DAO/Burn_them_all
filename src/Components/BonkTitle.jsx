const BonkTitle = () => {
  return (
    <div className="bonkTitleContainer">
      <div className="title">
        <p className="emText">
          BONK EM ALL
        </p>
        <p className="burnText">BURN THE BONK$</p>
      </div>
      <div>
        <img
          src={require("../assests/Images/BonkLogo.png")}
          alt="BonkLogo"
          className="bonkLogo"
        />
      </div>
    </div>
  );
};

export default BonkTitle;
