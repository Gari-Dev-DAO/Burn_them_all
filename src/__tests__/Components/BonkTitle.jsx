import { render } from "@testing-library/react";
import BonkTitle from "../../Components/BonkTitle";

describe("Bonk Title", () => {
  it("renders component", () => {
    const { container } = render(<BonkTitle />);
    expect(container).toMatchSnapshot();
  });
});