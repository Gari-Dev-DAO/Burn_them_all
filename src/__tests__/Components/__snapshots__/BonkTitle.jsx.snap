// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bonk Title renders component 1`] = `
<div>
  <div
    class="bonkTitleContainer"
  >
    <div
      class="title"
    >
      <p
        class="emText"
      >
        BONK EM ALL
      </p>
      <p
        class="burnText"
      >
        BURN THE BONK$
      </p>
    </div>
    <div>
      <img
        alt="BonkLogo"
        class="bonkLogo"
        src="BonkLogo.png"
      />
    </div>
  </div>
</div>
`;
